.eslintrc.json

```diff
@@ -0,0 +1,35 @@
+{
+    "plugins": ["prettier", "@typescript-eslint"],
+    "env": {
+        "es6": true,
+        "node": true
+    },
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
```

Comment:
If you want you can use your tsconfig here as the parserOption:

```json
"parserOptions": {
    "project": "./tsconfig.json"
},
```

Reply:
@thibmaek, thank you! I have updated the parserOptions